.devcontainer/cuda11.8-conda/devcontainer.json

```diff
@@ -5,9 +5,14 @@
     "args": {
       "CUDA": "11.8",
       "PYTHON_PACKAGE_MANAGER": "conda",
-      "BASE": "rapidsai/devcontainers:24.08-cpp-llvm16-cuda11.8-mambaforge-ubuntu22.04"
+      "BASE": "rapidsai/devcontainers:24.08-cpp-cuda11.8-mambaforge-ubuntu22.04"
```

Comment:
Guessing `llvm16` is no longer needed as the image contains LLVM 16 (or newer) now?

Reply:
Yes, a bit before this PR was opened we removed all LLVM packages from the mambaforge image, and all except `clang-format-16` and `clangd-dev` from the pip image.

In the conda case, the clang-tools should be installed via conda sections in `dependencies.yaml`, so ncluding them in the image is unnecessary. Additionally, the apt-installed `clangd` would break due to using the conda env's LLVM libs instead of the system versions.

In the pip case, we only keep `clang-format-16` and `clangd-dev`, again to cut down on image size. Since `clangd`'s CUDA support is nearly always playing catch-up with the latest CUDA releases, we install the dev branch so the latest intellisense features are available.